gammapy/cube/new.py

```diff
@@ -1,7 +1,7 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 from __future__ import absolute_import, division, print_function, unicode_literals
 import numpy as np
-# import astropy.units as u
+#import astropy.units as u
```

Comment:
Remove this commented out line. (IMO we should never leave old uncommented or unused code in the repo, it's just a distraction)